### PR TITLE
chore: update mediacloud url to civicsignal url

### PR DESCRIPTION
**Pull request by VinneyJ**
Updated the base api URLs from https://api.mediacloud.org/ to https://app.civicsignal.africa:8082/ in api_2_0_spec documention.

### CODE REVIEW

File: doc/api_2_0_spec/api_2_0_spec.md
Comment (kilemensi) [inline doc/api_2_0_spec/api_2_0_spec.md:258]: Aren't we removing all ports?

Comment (VinneyJ) [reply doc/api_2_0_spec/api_2_0_spec.md:258]: @thepsalmist mentioned that the ports are exposed to the frontend apps. @kilemensi

Comment (kilemensi) [reply doc/api_2_0_spec/api_2_0_spec.md:258]: Cool @VinneyJ ... my question is do we want to do this moving forward or not?

Comment (VinneyJ) [reply doc/api_2_0_spec/api_2_0_spec.md:258]: No, @kilemensi, this is not the ideal approach. I recommend changing the port exposure. I will discuss this further with @thepsalmist to see how we can approach removing them.

**Review by kelvinkipruto — APPROVED** [review]
@VinneyJ LGTM once the exposed port issue is addressed.

**Comment by DavidTheProgrammer** [comment]
> @VinneyJ LGTM once the exposed port issue is addressed.

@VinneyJ same here.

**Comment by VinneyJ** [comment]
Hello @kilemensi / @thepsalmist since I cloned the repo again let's merge this then I bring in the new changes. What do you say?

**Comment by kilemensi** [comment]
It's one file @VinneyJ, isn't it easier to copy it to the new repo and create a new PR?

**Comment by VinneyJ** [comment]
Yes @kilemensi that can be another approach.

**Comment by DavidTheProgrammer** [comment]
is this PR still valid seeing as we have the updated one? @VinneyJ

**Comment by kilemensi** [comment]
Nah, this should automatically be closed once #4 is merged.

**Pull request closed without merging**
